ExamGenerator.Web.InputModels/QuestionType/QuestionTypeInputModel.cs

```diff
@@ -0,0 +1,14 @@
+﻿using ExamGenerator.Services.Mapping;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace ExamGenerator.Web.InputModels.QuestionType
+{
+    public class QuestionTypeInputModel:IMapFrom<Data.Models.QuestionType>
```

Comment:
Should be IMapTo because you want to map from QuestionTypeInputModel to QuestionType